src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,572 @@
+  },
+  false,
+);
+```
+
+The console output will depend on which element you click. If you were to click on the "deepest"
+element in the DOM tree (the `#C` element), you will see every single one of these event handlers
+fire. With a bit of CSS styling to make it more obvious which element is which, here is the console
+output `#C` element (with a screenshot as well):
+
+```bash
+"click on document in capturing phase"
+"click on <html> in capturing phase"
+"click on <body> in capturing phase"
+"click on #A in capturing phase"
+"click on #B in capturing phase"
+"click on #C in capturing phase"
+"click on #C in bubbling phase"
+"click on #B in bubbling phase"
+"click on #A in bubbling phase"
+"click on <body> in bubbling phase"
+"click on <html> in bubbling phase"
+"click on document in bubbling phase"
+```
+
+You can interactively play with this in the live demo below. Click on the `#C` element and observe the console output.
+
+<div style="height: 680px; width: 100%;">
+  <iframe src="https://silicon-brawny-cardinal.glitch.me/event-capturing-and-bubbling.html" loading="lazy" style="height: 100%; width: 100%; border: 0;"></iframe>
+</div>
+
+### `event.stopPropagation()`
+
+With an understanding of where events originate and how they travel (i.e. propagate) through the DOM
+in both the capturing phase and the bubbling phase, we can now turn our attention to
+`event.stopPropagation()`.
+
+The `stopPropagation()` method can be called on (most) native DOM events. I say "most" because there
+are a few on which calling this method won't do anything (because the event doesn't propagate to
+begin with). Events like `focus`, `blur`, `load`, `scroll`, and a few others fall into this
+category. You can call `stopPropagation()` but nothing interesting will happen, since these events
+don't propagate.
+
+### But what does stopPropagation do?
+
+It does, pretty much, just what it says. When you call it, the event will, from that point, cease
+propagating to any elements it would otherwise travel to. This is true of _both_ directions
+(capturing and bubbling). So if you call `stopPropagation()` anywhere in the capturing phase, the
+event will never make it to the target phase or bubbling phase. If you call it in the bubbling
+phase, it will have already gone through the capturing phase, but it will cease "bubbling up" from
+the point at which you called it.
+
+Returning to our same example markup, what do you think would happen, if we called
+`stopPropagation()` in the capturing phase at the `#B` element?
+
+It would result in the following output:
+
+```bash
+"click on document in capturing phase"
+"click on <html> in capturing phase"
+"click on <body> in capturing phase"
+"click on #A in capturing phase"
+"click on #B in capturing phase"
+```
+
+You can interactively play with this in the live demo below. Click on the `#C` element in the live demo and observe the console output.
+
+<div style="height: 680px; width: 100%;">
```

Comment:
We have a Glitch embed. I don't mind you using this if you need it, but is there a reason that embed isn't working for you?

Reply:
This was me. In this case I wanted the user to be immersed into the experience completely and not distracted by the Glitch surrounding, so I chose to embed the iframes directly.